catalog.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Техномарт</title>
+  </head>
+  <body>
+
+    <header>
+        <a>
+          <img src="https://via.placeholder.com/350x65" width="150" alt="логотип Техномарт">
+        </a>
+        <input type="search" placeholder="Поиск:">
```

Comment:
Раз уж установил элемент формы, то и саму форму поставь. Без нее поиск работать не будет
```suggestion
        <form action="search.html">
            <input type="search" placeholder="Поиск:">
        </html>
```